delfin/api/v1/router.py

```diff
@@ -62,19 +62,28 @@ def _setup_routes(self, mapper):
                        action="update",
                        conditions={"method": ["PUT"]})
 
+        mapper.connect("storages", "/access-infos",
+                       controller=self.resources['access_info'],
+                       action="show_all",
+                       conditions={"method": ["GET"]})
+
         self.resources['alert_sources'] = alert_source.create_resource()
-        mapper.connect("storages", "/storages/{id}/alert-source",
+        mapper.connect("storages", "/storages/{id}/snmp-config",
```

Comment:
Can you update this changes in API spec swagger file also  /openapi-spec/swagger.yml

Reply:
Added